templates/api/knowledge-panels/health/nutrition/nutrient_level.tt.json

```diff
@@ -0,0 +1,142 @@
+{
+    "level": "info",
+    "topics": [
+        "health"
+    ],
+    "size": "small",
+    "evaluation": "[% panel.evaluation %]",
+    "title_element": {
+        "title": "[% panel.nutrient_in_quantity %] ([% panel.nutrient_quantity_in_grams %]%)",
+        "icon_url": "[% static_subdomain %]/images/misc/[% panel.nutrient_level %].svg",
+        "icon_size": "small",
+    },
+    "elements": [
+        // Explanation of the impact of the nutrient
+        // + recommendations (e.g. reduce)
+        // based on WHO recommendations, national health agencies etc.
+        [% IF panel.nid == 'salt' %]
+        {
+            "element_type": "text",
+            "text_element": {
+                "type": "summary",
+                    "html":
+`
+<b>[% lang("knowledge_panels_facts") %]</b>
+<ul>
+<li>[% lang('nutrient_info_salt_risk') %]</li>
+<li>[% lang('nutrient_info_salt_high_blood_pressure') %]</li>
+<li>[% lang('nutrient_info_salt_high_consumption') %]</li>
+</ul>
+<br>
+<b>[% lang("knowledge_panels_recommendation") %]</b>[% sep %]:
+<b>[% lang('nutrient_recommendation_salt_title') %]</b>
+<ul>
+<li>[% lang('nutrient_recommendation_salt_cooking_and_table') %]</li>
+<li>[% lang('nutrient_recommendation_salt_limit_salty_snacks') %]</li>
+</ul>
+`,                  
```

Comment:
I find it a bit annoying that this is defined here so deeply in a technical construct. Shouldn't we put it in an external template (that we could include here) ?

Reply:
I'm not really sure if a new level of indirection would make things simpler or more complex. The initial idea was that the knowledge panels templates was the place where we could define what information is presented and how.

I'll merge as-is and we can refine